pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -188,7 +188,8 @@ async def _trigger_microshift_sync(self):
         major, minor = self._ocp_version
         version = f'{major}.{minor}'
         try:
-            jenkins.start_microshift_sync(version=version, assembly=self.assembly, dry_run=self.runtime.dry_run)
+            jenkins.start_microshift_sync(version=version, assembly=self.assembly,
+                                          dry_run=self.runtime.dry_run, block_until_complete=True)
```

Comment:
what dose the `block_until_complete` used for?

Reply:
It blocks the parent job until this child job is complete